### DISCUSSION TITLE
Link variable - logging out issue

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14978] [How to] [answered]
I have little thing with links generated by variable in email templates/workflows.
The link to record comes out in the format such as: .../index.php?module=HelpDesk&view=Detail&record=43510
Clicking link killing user session logging out of all tabs and forcing to log again. 
Is there and setting in yetiforce to prevent that?

Appreciate tips

**Comment by vovpff** [comment, marked as answer]
I has faced the sane problem but with sharing URLs by messengers. There is some parameters in configs available for cookies. I can't reach my server to check what parameter you need to modify.

Try to find it by youself.

**Comment by s2b-git** [comment]
Security.php > public static $cookieSameSite = 'Lax';

Above works. Thanks
